horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/health/HorreumHealthCheck.java

```diff
@@ -0,0 +1,15 @@
+package io.hyperfoil.tools.horreum.svc.health;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import org.eclipse.microprofile.health.HealthCheck;
+import org.eclipse.microprofile.health.HealthCheckResponse;
+import org.eclipse.microprofile.health.Readiness;
+
+@Readiness
+@ApplicationScoped
+public class HorreumHealthCheck implements HealthCheck {
+   @Override
+   public HealthCheckResponse call() {
+      return HealthCheckResponse.up("Horreum is ready to accept requests");
```

Comment:
This could be improved to check the authentication server is responding or anything else